### ISSUE TITLE
triv. typo

### ISSUE THREAD

**Issue by tbnorth** [leo-editor/leo-editor#373]
in docs. in table of handlers:

```
‘select1’	yes	before selecting a position	c,new_p,old_p,new_v,new_v
```

should not list new_v twice

**Comment by edreamleo** [comment]
Fixed at 1f1e5cb, and on the web.

**Comment by edreamleo** [comment]
On Sat, Jan 21, 2017 at 1:57 PM, Terry Brown <notifications@github.com>
wrote:

> in docs. in table of handlers:
>
> ‘select1’	yes	before selecting a position	c,new_p,old_p,new_v,new_v
>
> should not list new_v twice
>

​Fixed at 1f1e5cb, and on the web.  The second new_v is now old_v, just
like the other entries.

EKR
